instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op5",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_frontal-operculum"
+        }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "JBA_Area-Op5",
```

Comment:
this should be OP5 (as it was OP1 - OP4)

Reply:
please see https://github.com/HumanBrainProject/openMINDS_SANDS/pull/151#discussion_r765939025